src/auction.js

```diff
@@ -593,6 +596,17 @@ function tryAddVideoBid(auctionInstance, bidResponse, afterBidAdded, {index = au
   }
 }
 
+// Native bid response might be in ortb2 format - adds legacy field for backward compatibility
+function addLegacyFieldsIfNeeded(bidResponse) {
+  const nativeOrtbRequest = auctionManager.index.getBidRequest(bidResponse)?.nativeOrtbRequest;
```

Comment:
this should be `index.getAdUnit` - in some situations you can have a bid response without a bid request.